presto-hive/src/test/java/com/facebook/presto/hive/HiveQueryRunner.java

```diff
@@ -75,8 +75,20 @@ public static DistributedQueryRunner createQueryRunner(Iterable<TpchTable<?>> ta
         return createQueryRunner(tables, extraProperties, "sql-standard", ImmutableMap.of());
     }
 
+    public static DistributedQueryRunner createQueryRunner(Iterable<TpchTable<?>> tables, Map<String, String> extraProperties, Optional<String> baseDirString)
```

Comment:
Use `File` to pass arguments. Converting from a string to a path should be a concern for the caller. APIs in general should use appropriate types rather than strings.